src/Libraries/Microsoft.Extensions.AI.Evaluation/EvaluatorExtensions.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.AI.Evaluation;
+
+/// <summary>
+/// Extension methods for <see cref="IEvaluator"/>.
+/// </summary>
+public static class EvaluatorExtensions
+{
+    private static IEnumerable<ChatMessage> EmptyMessages { get; } = [];
```

Comment:
Same question as earlier

Reply:
I've gone ahead and replaced all usages of `EmptyMessages` with `[]`.